sdk/keyvault/keyvault-keys/src/generated/keyVaultClient.ts

```diff
@@ -392,18 +392,20 @@ class KeyVaultClient extends KeyVaultClientContext {
    * @param keyName The name of the key.
    * @param keyVersion The version of the key.
    * @param algorithm algorithm identifier. Possible values include: 'RSA-OAEP', 'RSA-OAEP-256',
-   * 'RSA1_5'
+   * 'RSA1_5', 'A128GCM', 'A192GCM', 'A256GCM', 'A128KW', 'A192KW', 'A256KW', 'A128CBC', 'A192CBC',
+   * 'A256CBC', 'A128CBCPAD', 'A192CBCPAD', 'A256CBCPAD'
    * @param value
    * @param [options] The optional parameters
    * @returns Promise<Models.EncryptResponse>
    */
-  encrypt(vaultBaseUrl: string, keyName: string, keyVersion: string, algorithm: Models.JsonWebKeyEncryptionAlgorithm, value: Uint8Array, options?: coreHttp.RequestOptionsBase): Promise<Models.EncryptResponse>;
+  encrypt(vaultBaseUrl: string, keyName: string, keyVersion: string, algorithm: Models.JsonWebKeyEncryptionAlgorithm, value: Uint8Array, options?: Models.KeyVaultClientEncryptOptionalParams): Promise<Models.EncryptResponse>;
```

Comment:
This isn't a breaking change?

Reply:
This is the generated code! The public API changes are here: https://github.com/Azure/azure-sdk-for-js/pull/11370/files#diff-91e6187853dc6c6c676058faaa81d8e2L212